webapp/channels/src/components/help/formatting.tsx

```diff
@@ -0,0 +1,719 @@
+                        />
+                    </h2>
+                    <p>
+                        <FormattedMessage
+                            id='help.formatting.code_block.description'
+                            defaultMessage='Create a code block by indenting each line by four spaces, or by placing ``` on the line above and below your code.'
+                        />
+                    </p>
+                    <p>
+                        <FormattedMessage
+                            id='help.formatting.code_block.example_label'
+                            defaultMessage='Example:'
+                        />
+                    </p>
+                    <table className='Help__table'>
+                        <thead>
+                            <tr>
+                                <th>
+                                    <FormattedMessage
+                                        id='help.formatting.table.text_entered'
+                                        defaultMessage='Text Entered'
+                                    />
+                                </th>
+                                <th>
+                                    <FormattedMessage
+                                        id='help.formatting.table.how_it_appears'
+                                        defaultMessage='How it appears'
+                                    />
+                                </th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr>
+                                <td>
+                                    <code>{'```'}</code><br/>
+                                    <code>{'Code block'}</code><br/>
+                                    <code>{'```'}</code>
+                                </td>
+                                <td>
+                                    <pre className='Help__code-preview'>{'Code block'}</pre>
+                                </td>
+                            </tr>
+                        </tbody>
+                    </table>
+                </section>
+
+                <section className='Help__section'>
+                    <h2>
+                        <FormattedMessage
+                            id='help.formatting.syntax.title'
+                            defaultMessage='Syntax Highlighting'
+                        />
+                    </h2>
+                    <p>
+                        <FormattedMessage
+                            id='help.formatting.syntax.description'
+                            defaultMessage='To add syntax highlighting, type the language to be highlighted after the ``` at the beginning of the code block. Mattermost also offers four different code themes (GitHub, Solarized Dark, Solarized Light, Monokai) that can be changed in <b>Settings > Display > Theme > Custom Theme > Center Channel Styles > Code Theme</b>.'
```

Comment:
```suggestion
                            defaultMessage='To add syntax highlighting, type the language to be highlighted after the <code>```</code> at the beginning of the code block. Mattermost also offers four different code themes (GitHub, Solarized Dark, Solarized Light, Monokai) that can be changed in <b>Settings > Display > Theme > Custom Theme > Center Channel Styles > Code Theme</b>.'
```